crates/rpc-types/src/eth/admin.rs

```diff
@@ -0,0 +1,409 @@
+//! Types for the admin api
+use alloy_genesis::ChainConfig;
+use alloy_primitives::{B256, U256};
+use alloy_serde::json_u256::deserialize_json_u256;
+use serde::{Deserialize, Serialize};
+use std::net::{IpAddr, SocketAddr};
+
+/// This includes general information about a running node, spanning networking and protocol
+/// details.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct NodeInfo {
+    /// The node's private key.
+    pub id: B256,
```

Comment:
this is the wrong type, should be B512

Reply:
Yes, you are right. I checked both reth and web3.py are using B512. 

https://web3py.readthedocs.io/en/stable/web3.geth.html#web3.geth.admin.node_info

Ethers-rs is using H256 here, so I assume it should be updated?
https://github.com/gakonst/ethers-rs/blob/master/ethers-providers/src/ext/admin.rs#L14

Reply:
I see, for the pr we need to figure out what this is set to